source/app/ImmuneMLApp.py

```diff
@@ -15,6 +28,49 @@
 from source.util.ReflectionHandler import ReflectionHandler
 
 
+_DATE_FMT = '%d.%m.%Y %H:%M:%S'
+
+
+def hello_world(infile, rundir, outfile):
+    """Print out a politically correct greeting for immuneML.
+
+    Parameters
+    ----------
+    infile : string
+        String showing the location of the input file.
+    rundir : string
+        String showing the location we are running in.
+    outfile : string
+        The output file
+
+    """
+    timestart = datetime.datetime.now().strftime(_DATE_FMT)
+    pyversion = sys.version.split()[0]
+    print('\n'.join([LOGO]))
+    print(f'{PROGRAM_NAME} version: {VERSION}')
+    print(f'Start of execution: {timestart}')
+    print(f'Python version: {pyversion}')
+    print(f'Running in directory: {rundir}')
+    print(f'Input file: {infile}')
+    print(f'Output file: {outfile}')
```

Comment:
result_path is not a file, so this is a bit confusing

Reply:
fixed